test/com/stanimiriliev/javaobjects/sumator/SumatorTest.java

```diff
@@ -0,0 +1,37 @@
+package com.stanimiriliev.javaobjects.sumator;
```

Comment:
It is in separate directory. I do not know why this had happen.
![image](https://user-images.githubusercontent.com/15329619/29074068-a0f9b8dc-7c56-11e7-934f-2af49142784c.png)